<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CLOCK</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="screen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>12:00:00 AM</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="changeFG">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CHANGE FOREGOUND COLOR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="changeBG">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>90</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>CHANGE BACKGROUND COLOR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorEntry">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter color name or hexa-color code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyTheme">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>APPLY</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fontEntry">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter font style name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="applyFont">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>230</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>APPLY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fontStyleButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CHANGE FONT STYLE</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
